docs/develop/overview-authn-authz.md

```diff
@@ -7,7 +7,7 @@ localization_priority: Priority
 
 # Overview of authentication and authorization in Office Add-ins
 
-Web applications and, hence, Office Add-ins allow anonymous access by default, but you can require users to authenticate with a login. In particular, you can require that your users be logged in with either a Microsoft Account or a Work or School (Office 365) account. This task is called user authentication because it enables the add-in to know who the user is.
+Web applications and, hence, Office Add-ins allow anonymous access by default, but you can require users to authenticate with a login. For example, you can require that your users be logged in with either a Microsoft Account or a Work or School (Office 365) account, or other common accounts. This task is called user authentication because it enables the add-in to know who the user is.
```

Comment:
This sentence is a little awkward with the "either... or … or". Consider:
"For example, you can require that your users be logged in with a Microsoft Account, an Office 365 Work or School account, or another common account."